### DISCUSSION TITLE
Expired Discord Invite

### DISCUSSION THREAD

**Discussion by mcDevnagh** [Obsidian-TTRPG-Community/TTRPG-Community-Admin#31] [Ask a Sourcerer] [answered]
The link for the Organization ([https://discord.gg/Zpmr37Uv](https://discord.gg/Zpmr37Uv)) is expired

**Comment by sigrunixia** [comment, marked as answer]
Thanks for bringing this to our attention. It has been updated to https://discord.gg/pCwMjBE6pY.
